server/controllers/medical/reports/checkins.js

```diff
@@ -5,40 +5,24 @@
  * This file contains code to create a PDF report of all patient checkins,
  * matching query conditions passed from the patient registry UI grid.
  *
- * @requires path
  * @requires lodash
- * @requires BadRequest
- * @requires db
- * @requires renderers/json
- * @requires renderers/html
- * @requires renderers/pdf
+ * @requires lib/db
+ * @requires lib/ReportManager
+ * @requires Patients
```

Comment:
Could we have a good name for `checkin` in the french transaction
![chekin](https://cloud.githubusercontent.com/assets/5445251/19345011/7151c616-90f1-11e6-8d3c-747baae1e81c.png)

`Verification des patients` is not very clear for the action to do.